Command/EventLogCleanupCommand.php

```diff
@@ -82,10 +86,19 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             EventLogCleanup::CAMPAIGN_LEAD_EVENTS => $input->getOption('campaign-lead'),
             EventLogCleanup::LEAD_EVENTS          => $input->getOption('lead'),
             EventLogCleanup::EMAIL_STATS          => $input->getOption('email-stats'),
+            EventLogCleanup::EMAIL_STATS_TOKENS   => $input->getOption('email-stats-tokens')
         ];
 
         if (0 === array_sum($operations)) {
             $operations = array_combine(array_keys($operations), array_fill(0, count($operations), true));
+            array_pop($operations);
+            $operations["email_stats_tokens"] = false;
+        }
+
+        if (($operations["email_stats_tokens"] === true) && ((($operations["email_stats"] === true) || ($operations["campaign_lead_event_log"] === true)) || ($operations["lead_event_log"] === true))) {
```

Comment:
Same here: use the constants as array keys

Reply:
Thanks @moongazer for the recommendation to better use constans if we have them - changed this not harmful part in the better practise part with my current commit.